src/main.py

```diff
@@ -0,0 +1,156 @@
+import re
+import serial
+from statistics import mean
+from datetime import datetime
+from typing import List
+from dotenv import load_dotenv
+import contextlib
+from dataclasses import dataclass
+
+
+load_dotenv()
+
+# Settings
+import os
+database_type = os.getenv("db.type")
+database_host = os.getenv("db.hostname")
+database_port = int(os.getenv("db.port"))
+database_username = os.getenv("db.username")
+database_password = os.getenv("db.password")
+database_database = os.getenv("db.database")
```

Comment:
Isn't the general convention to use upper case with underscores? Or is there some automatic mapping going on here which makes it work with, for example, `DB_HOSTNAME`?

Reply:
discussed. by using `.` notation we can use yaml config in a later stage/now in dev.

__it also just looks nice__